### PR TITLE
Add new organisation schema

### PR DESCRIPTION
**Pull request by rubenarakelyan**
This commit adds a new organisastion schema that will be used when populating organisations in the content-store. This will allow everything that is currently represented on these pages to be stored in the content-store. It also deprecates a link type in the placeholder schema that will be replaced by
an equivalent in the organisation schema.

Trello: https://trello.com/c/xebVLRcp/24-work-out-what-information-we-need-to-publish-to-the-content-store-for-organisations-and-add-a-schema-for-it

### CODE REVIEW

File: formats/placeholder.jsonnet [hunk 1]
```diff
@@ -53,13 +70,423 @@
...
+              },
+              image_href: {
+                type: "string",
+                format: "uri",
+              },
+            },
+          },
+          description: "Used for organisations, to allow us to publish board member information. Remove when organisations are migrated.",
+        },
+        ordered_military_personnel: {
```
Comment (Davidslv) [inline formats/placeholder.jsonnet]: Do we have intentions of extracting this into another schema in the future?

Comment (rubenarakelyan) [reply formats/placeholder.jsonnet]: I think once we migrate people and roles then we'll make these links in the links hash so they get updated automatically using dependency resolution.

File: formats/organisation.jsonnet [hunk 2]
```diff
@@ -0,0 +1,571 @@
...
+                null,
+              ],
+            },
+            image: {
+              "$ref": "#/definitions/image",
+            },
+          },
+          description: "The organisation's logo, including the logo image and formatted name.",
+        },
+        ordered_corporate_information_pages: {
```
Comment (tijmenb) [inline formats/organisation.jsonnet]: I think `ordered_corporate_information_pages` and `ordered_featured_links` are all migrated, could they go into the edition links?

File: formats/organisation.jsonnet [hunk 3]
```diff
@@ -0,0 +1,571 @@
...
+                "$ref": "#/definitions/image",
+              },
+              unpaid: {
+                type: "boolean",
+              },
+            },
+          },
+          description: "A list of the organisation's special representatives, if appropriate. Turn into proper links once organisations are fully migrated.",
+        },
+        ordered_contacts: {
```
Comment (tijmenb) [inline formats/organisation.jsonnet]: I think these also might be migrated, not sure though.

File: formats/organisation.jsonnet [hunk 4]
```diff
@@ -0,0 +1,571 @@
...
+                "$ref": "#/definitions/image",
+              },
+            },
+          },
+          description: "A set of links to social media profiles for the organisation.",
+        },
+        external_related_links: {
+          "$ref": "#/definitions/external_related_links",
+        },
+        tags: {
```
Comment (tijmenb) [inline formats/organisation.jsonnet]: What are `tags` for?

File: formats/organisation.jsonnet (hunk 2 above)
Comment (rubenarakelyan) [reply formats/organisation.jsonnet]: They are but they can also be external links so we need the flexibility...

File: formats/organisation.jsonnet (hunk 4 above)
Comment (rubenarakelyan) [reply formats/organisation.jsonnet]: Hangover from copying the placeholder schema - removed

File: formats/organisation.jsonnet (hunk 2 above)
Comment (tijmenb) [reply formats/organisation.jsonnet]: Ah that's unfortunate. We should make a note to represent the external links too ([see Mat's RFC](https://github.com/alphagov/govuk-rfcs/blob/master/rfc-088-external-content.md)) 

Does this apply to `ordered_corporate_information_pages` as well?

File: formats/organisation.jsonnet (hunk 3 above)
Comment (rubenarakelyan) [reply formats/organisation.jsonnet]: Looks like they are published to the publishing-api, so moving them to links

File: formats/organisation.jsonnet (hunk 2 above)
Comment (rubenarakelyan) [reply formats/organisation.jsonnet]: Yeah both `ordered_corporate_information_pages` and `ordered_featured_links` can contain external links - for example to the Civil Service Jobs site or organisation-specific services on other sites.

**Pull request merged**